src/components/editor/editor-pane/tool-bar/tool-bar.tsx

```diff
@@ -40,9 +41,8 @@ export interface ToolBarProps {
 export const ToolBar: React.FC<ToolBarProps> = ({ editor }) => {
   const { t } = useTranslation()
 
-  const notImplemented = () => {
-    alert('This feature is not yet implemented')
-  }
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { clickOnHiddenFileInput, HiddenFileInput } = useFilePicker(editor)
```

Comment:
why are you using a hook instead of a component?

Reply:
Because the "component" does not render anything. It's just a function (that also happens to need a hidden input)

Reply:
That's wrong. It renders a hidden file input.

Reply:
Fine, if you insist.